src/helpers.ts

```diff
@@ -252,3 +252,55 @@ export function hasMinutesPassed(
     new Date().getTime() - new Date(timeToCheck).getTime() > minutes * 60 * 1000
   );
 }
+
+/**
+ * This TypeScript function converts a given date string into a formatted date and time string in UTC
+ * timezone.
+ * @param {string} date - The `convertDateTimelinePointDate` function takes a `date` parameter as a
+ * string input. This date parameter represents a timestamp that needs to be converted into a specific
+ * format using the `toLocaleString` method with the provided options.
+ * @returns The function `convertDateTimelinePointDate` takes a date string as input, converts it to a
+ * Date object, and then formats it using the specified options. The formatted date string is returned.
+ */
+export const convertDateTimelinePointDate = (date: string): string => {
+  if (!date) return '';
+  const timestamp = new Date(date);
+
+  const options: Intl.DateTimeFormatOptions = {
+    year: 'numeric',
+    month: 'short',
+    day: '2-digit',
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: true,
+    timeZone: 'UTC',
+  };
+
+  return timestamp.toLocaleString('en-US', options);
+};
+
+/**
```

Comment:
This might be a personal opinion, but these long paragraphs seem to be overexplaining the code